conftest.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Configuration of pytest
+"""
+
+import pytest
+from django.conf import settings
```

Comment:
E0401 - Unable to import 'django.conf'

*Origin: PyLintBear (E0401), Section: `all.python`.*